src/hooks/useConnectors.ts

```diff
@@ -0,0 +1,31 @@
+import { TABLES } from 'services/supabase';
+import { useQuery, useSelect } from './supabase-swr/';
+
+interface Connector {
+    id: string;
+    title: { 'en-US': string };
+    image_name: string;
+}
+
+export const CONNECTOR_QUERY = `
+    id, title, image_name
+`;
+
+function useConnectors() {
```

Comment:
A hook for fetching `connectors` directory from their own table, without any association with `connector_tags`

Reply:
Added this comment into the code to make sure this knowledge wasn't lost